tools/mo/unit_tests/mo/utils/ir_reader/ops_test.py

```diff
@@ -0,0 +1,123 @@
+# Copyright (C) 2018-2023 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+import unittest
+import tempfile
+import numpy as np
+from pathlib import Path
+from openvino.tools.mo.utils.ir_reader.restore_graph import restore_graph_from_ir, save_restored_graph
+
+import openvino.runtime.opset11 as opset11
+import openvino.runtime.opset10 as opset10
+from openvino.runtime import Model, serialize
+
+
+class TestOps(unittest.TestCase):
+    def test_topk_11(self):
+        data_shape = [6, 12, 10, 24]
+        data_parameter = opset11.parameter(
+            data_shape, name="Data", dtype=np.float32)
+        k_val = np.int32(3)
+        axis = np.int32(1)
+        topk = opset11.topk(data_parameter, k_val, axis,
+                            "max", "value", stable=True, name="TopK_11")
+        model = Model(topk, [data_parameter])
+        with tempfile.TemporaryDirectory() as tmp:
+            model_xml = Path(tmp) / 'topk_model.xml'
+            model_bin = Path(tmp) / 'topk_model.bin'
+            serialize(model, model_xml, model_bin)
+            graph, _ = restore_graph_from_ir(model_xml, model_bin)
+            topk_node = graph.nodes()["TopK_11"]
+            self.assertEqual(topk_node["version"], "opset11")
+            self.assertTrue(topk_node["stable"])
+
+    def test_interpolate_11(self):
+        data_shape = [6, 12, 10, 24]
+        data_parameter = opset11.parameter(
+            data_shape, name="Data", dtype=np.float32)
+        interpolate = opset11.interpolate(data_parameter, np.int32(
+            [20, 48]), "nearest", "sizes", axes=np.int32([2, 3]), name="Interpolate_11")
+        model = Model(interpolate, [data_parameter])
+        with tempfile.TemporaryDirectory() as tmp:
+            model_xml = Path(tmp) / 'interpolate_model.xml'
+            model_bin = Path(tmp) / 'interpolate_model.bin'
+            serialize(model, model_xml, model_bin)
+            graph, _ = restore_graph_from_ir(model_xml, model_bin)
+            interpolate_node = graph.nodes()["Interpolate_11"]
+            self.assertEqual(interpolate_node["version"], "opset11")
+
+    def test_interpolate_4(self):
+        data_shape = [6, 12, 10, 24]
+        data_parameter = opset10.parameter(
+            data_shape, name="Data", dtype=np.float32)
+        interpolate = opset10.interpolate(data_parameter, np.int32([20, 48]), np.float32(
```

Comment:
opset4?

Reply:
opset4 doesn't work for some reason, not sure why, but it can be because Interpolate-4 was delayed and didn't get into it